zone/questmgr.cpp

```diff
@@ -2580,6 +2580,32 @@ int QuestManager::collectitems(uint32 item_id, bool remove)
 	return quantity;
 }
 
+int QuestManager::countitem(uint32 item_id) {
+	QuestManagerCurrentQuestVars();
+	int quantity = 0;
+	EQEmu::ItemInstance *item = nullptr;
+	static const int16 slots[][2] = {
+		{ EQEmu::invslot::POSSESSIONS_BEGIN, EQEmu::invslot::POSSESSIONS_END },
+		{ EQEmu::invbag::GENERAL_BAGS_BEGIN, EQEmu::invbag::GENERAL_BAGS_END },
+		{ EQEmu::invbag::CURSOR_BAG_BEGIN, EQEmu::invbag::CURSOR_BAG_END},
+		{ EQEmu::invslot::BANK_BEGIN, EQEmu::invslot::BANK_END },
```

Comment:
Do we always want to check bank here?

I would imagine most use cases care only about what is in the current inventory

Reply:
You can use quest::collectitems(itemid, remove?) if you want just in inventory.

This method is specifically to count any/all occurrences of an item in a player's inventory.

Reply:
Makes sense!